src/main/java/org/folio/rest/impl/PurchaseOrderLineHelper.java

```diff
@@ -361,7 +361,9 @@ private void updateCreateInventory(CompositePoLine compPOL, JsonObject jsonConfi
         .thenCompose(compOrder -> {
           validatePOLineProtectedFieldsChanged(compOrderLine, lineFromStorage, compOrder);
           return protectionHelper.isOperationRestricted(compOrder.getAcqUnitIds(), UPDATE)
-            .thenCompose(vVoid -> validateAndNormalizeISBN(compOrderLine))
+            .thenCompose(v -> validateAndNormalizeISBN(compOrderLine))
+            .thenCompose(v -> new VendorHelper(httpClient, okapiHeaders, ctx, lang)
+              .validateAccessProviders(Collections.singletonList(compOrderLine)))
```

Comment:
I think we need test to check that accessProvider validation is performed during a poLine update